pcsx2/gui/MainFrame.cpp

```diff
@@ -665,7 +665,7 @@ void MainEmuFrame::ApplyCoreStatus()
 		}
 	}
 
-	menubar.Enable( MenuId_Sys_Shutdown, SysHasValidState() || CorePlugins.AreAnyInitialized() );
+	menubar.Check( MenuId_Sys_Shutdown, SysHasValidState() || CorePlugins.AreAnyInitialized() );
```

Comment:
> I guess it's possible, (looking at the Wx Reference ) but then why doesn't this function correctly with menubar.Enable ? I though this worked at one point, but I can't recall

Well, ` SysHasValidState() || CorePlugins.AreAnyInitialized()` must be returning `true`.
 
> Edit: maybe there is a bug in Wx? People here seem to note that menubar.Check behaves differently on certain platforms. It's not defined in the documentation specifically (IMO) but isn't this just checking if it was clicked? (not a physical checkbox?)

`Check()` changes whether a checkmark is present.. The documentation is pretty clear IMO.